xinference/core/tests/test_restful_api.py

```diff
@@ -356,6 +356,7 @@ def test_restful_api_for_embedding(setup):
 
     assert "embedding" in embedding_res["data"][0]
     assert len(embedding_res["data"][0]["embedding"]) == model_spec.dimensions
+    assert "model_replica" in embedding_res.keys()
```

Comment:
```
assert embedding_res["model_replica"] is not None
```

Not only to check if the key exists, but also check if the value exists in real.

And, did you check the `model` ? That's the original problem to solve, to make sure `model` is identical to model uid.